tumpasrc/__init__.py

```diff
@@ -279,6 +339,10 @@ def generate(self):
             subkeys_expiration=True,
         )
         self.update_ui.emit(newk)
+        success_dialog = QtWidgets.QMessageBox()
+        success_dialog.setWindowTitle("New key generated")
+        success_dialog.setText("Key generation successful!")
+        success_dialog.exec()
```

Comment:
This dialog is missing the stylesheet. Also, you can reuse `show_success_dialog` method I think.

![Screenshot from 2020-12-28 12-00-15](https://user-images.githubusercontent.com/272303/103194111-82e9f680-4904-11eb-89dc-7170239e0e32.png)

Reply:
Can't reuse that function because it is part of MainWindow, and not dialog. And since there was only one instance of using success diaog in the key generation dialog, I didn't implement a separate function (for now)

Reply:
Added the stylesheet